### PR TITLE
Adding new TOC entries.

### PR DESCRIPTION
**Pull request by egilmorez**
Staged and discussed internally in the known places.

Thanks!

### CODE REVIEW

File: docgen/content-sources/v1/toc.yaml [hunk 1]
```diff
@@ -135,6 +135,20 @@ toc:
       - title: 'ObjectMetadata'
         path: /docs/reference/functions/providers_storage.objectmetadata.html
 
+  - title: functions.tasks
+    path: /docs/reference/functions/providers_tasks.html
+    section:
+    - title: common_providers_tasks.authdata.html
```
Comment (inlined) [inline docgen/content-sources/v1/toc.yaml:141]: These titles should probably be `AuthData`, `RateLimits`, `Request`, `RetryConfig`, and `TaskContext` respectively. Also, for v1 we don't have `Request`; data + `TaskContext` is v1 and `Request` is v2.

Comment (egilmorez) [reply docgen/content-sources/v1/toc.yaml:141]: Yup, sorry, that's how I fixed them up in the DevSite TOC.  

Will fix that and remove Request.

BTW how can I remove package-lock.json from this PR?  Neither revert nor checkout worked for me.  Thanks!

File: docgen/content-sources/v1/toc.yaml [hunk 2]
```diff
@@ -135,6 +135,18 @@ toc:
...
+    section:
+      - title: AuthData
+        path: /docs/reference/functions/common_providers_tasks.authdata.html
+      - title: RateLimits
+        path: /docs/reference/functions/common_providers_tasks.ratelimits.html
+      - title: RetryConfig
+        path: /docs/reference/functions/common_providers_tasks.retryconfig.html
+      - title: TaskContext
+        path: /docs/reference/functions/common_providers_tasks.taskcontext.html
+
```
Comment (jhuleatt) [inline docgen/content-sources/v1/toc.yaml:149]: Looks like `TaskQueueBuilder` is missing

File: docgen/content-sources/v1/toc.yaml [hunk 3]
```diff
@@ -124,6 +126,8 @@ toc:
         path: /docs/reference/functions/providers_remoteconfig.remoteconfiguser.html
       - title: 'TemplateVersion'
         path: /docs/reference/functions/providers_remoteconfig.templateversion.html
+      - title: 'Updatebuilder'
```
Comment (jhuleatt) [inline docgen/content-sources/v1/toc.yaml:129]:
```suggestion
      - title: 'UpdateBuilder'
```

File: docgen/content-sources/v1/toc.yaml (hunk 2 above)
Comment (egilmorez) [reply docgen/content-sources/v1/toc.yaml]: Should be back now, thanks!

**Review by jhuleatt — APPROVED** [review]

File: docgen/content-sources/v1/toc.yaml (hunk 3 above)
Comment (egilmorez) [reply docgen/content-sources/v1/toc.yaml]: Got it, thanks!

**Pull request merged**